targets/alert-cli/src/exportContributionAlerts.js

```diff
@@ -42,30 +39,54 @@ export async function exportContributionAlerts(changes) {
 
 /**
  *
- * @param {alerts.DilaNodeForDiff} node
+ * @param {import("@shared/types").ParseDilaReference} reference
+ * @param {alerts.DilaAlertChanges} nodes
  */
-function computeDiff(node) {
+function computeDiff(reference, { added, removed, modified }) {
+  const addedNode = added.find((node) => node.data.cid === reference.dila_cid);
+  if (addedNode) {
+    return { type: "added" };
+  }
+  const removedNode = removed.find(
+    (node) => node.data.cid === reference.dila_cid
+  );
+  if (removedNode) {
+    return { type: "removed" };
+  }
+
+  const modifiedNode = modified.find(
+    (node) => node.data.cid === reference.dila_cid
+  );
+
   const textFieldname =
-    node.context.containerId === "LEGITEXT000006072050" ? "texte" : "content";
-  const content = node.data[textFieldname] || "";
+    modifiedNode.context.containerId === "LEGITEXT000006072050"
```

Comment:
shouldnt we match `/^LEGITEXT\d+$/` ?

Reply:
I would maybe invert condition and use  `/^KALITEXT\d+$/" ? "content" : "text"`